cairo-libs/packages/ics23/src/specs.cairo

```diff
@@ -13,7 +13,7 @@ pub fn iavl_spec() -> ProofSpec {
         min_prefix_length: 4,
         max_prefix_length: 12,
         child_size: 33,
-        empty_child: "",
```

Comment:
Can you add a comment a source for these values?
https://github.com/cosmos/ics23/blob/a324422529b8c00ead00b4dcee825867c494cddd/rust/src/api.rs#L198

Reply:
[f74daee](https://github.com/informalsystems/ibc-starknet/pull/305/commits/f74daee8bead51626ebffca6b408695b94fd622c)